llvm/include/llvm/MC/CAS/MCCASObjectV1.h

```diff
@@ -302,6 +302,12 @@ struct DwarfSectionsCache {
   MCSection *DebugInfo;
   MCSection *Line;
   MCSection *Str;
+  MCSection *Abbrev;
+};
+
+struct AbbrevAndDebugSplit {
```

Comment:
This may seem overkill now, but later we will also have a third member: the CAS object representing the abbrev offsets node